src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/FailedDocUploadProcessor.java

```diff
@@ -87,16 +88,10 @@ private void processEnvelopes(String containerName, List<Envelope> envelopes)
 
         log.info("Processing {} failed documents for container {}", envelopes.size(), containerName);
 
-        try {
-            CloudBlobContainer container = cloudBlobClient.getContainerReference(containerName);
+        CloudBlobContainer container = cloudBlobClient.getContainerReference(containerName);
 
-            for (Envelope envelope : envelopes) {
-                processEnvelope(container, envelope);
-            }
-        } finally {
-            if (serviceBusHelper != null) {
-                serviceBusHelper.close();
```

Comment:
looks like `.close()` is now never called.

Reply:
Not for now .... hold your breath ;-)

Reply:
Removed as we use a single instance of queue client now